api/OBJECT-SIGNING.md

```diff
@@ -76,7 +76,16 @@ The controller has TLS and signing certs, and the device has its device cert plu
 
 When using the `register` message to onboard the sequence is:
 
-- device boots up, and first sends a GET for `ControllerCerts` retrieving the payload signing and intermediate certificates used by the controller
+- device boots up
+- device checks for presence of initial network configuration in this order of preference:
+  1. legacy override.json (config partition) or usb.json (USB stick)
+       - if present, the (network) config is loaded and applied, which should open connectivity with the controller
```

Comment:
Why not have the bootstrap.pbuf have the highest priority?

Reply:
I wanted it to have the highest priority but others had different opinion. Their concern is that existing customers that rely on override/usb.json might get confused. I tried to explain that bootstrap config file will not be present in older systems or in EVE images created using current methods, so it should not cause any problems.
I will switch the order and explain this internally again.

Reply:
I wonder if we should actually ignore override/usb.json (even later appearing by newly inserted USB stick) if this new mechanism has been used to bootstrap the device.
What I mean, is that if user decides to use this new method and builds device-specific eve image for his device, then legacy override/usb jsons should be probably permanently disabled on his device to avoid mixing the two approaches.
@eriknordmark WDYT?

Reply:
If we see that we have a pbuf file at the same time as a json file, then definitely ingore the latter.
Question is whether it is easy and robust to remember that the device has been configured with a pbuf (from /config or from a USB stick) and in that case ignore (with a noisy log including on console?) any future json override.

Reply:
Done (in the implementation, for which I will open PR soon).